images/microOS/config.sh

```diff
@@ -175,7 +175,7 @@ fi
 
 if [[ "$kiwi_profiles" == *"Ceph"* ]]; then
   pip install fastapi uvicorn \
-    git+https://github.com/jecluis/aetcd3/@wip-aquarium#egg=aetcd3
```

Comment:
nit: I think using tags is more suitable than weird sha1.

Reply:
I agree, but I don't think we should be tagging the repo if we keep consuming from aetcd3's upstream, or we could end up with conflicting tags, or just making the whole history weird. Also, tagging just because one wants to merge a PR seems weirdly silly.